### ISSUE TITLE
tools.run_in_windows_bash causing unexpected keyword 'win_bash'

### ISSUE THREAD

**Issue by yeswalrus** [conan-io/conan#2855]
Using conan 1.3.1 on windows 10 with git for windows 2.17.0, the conan build step fails with the following output:
```
conan build . --source-folder=tmp/source --install-folder=tmp/install --build-folder=tmp/build
Project: Running build()
Project: run_in_windows_bash: "C:\Program Files\Git\usr\bin\bash.exe" --login -c ^"cd \^"/c/dev/external-libraries/conan/libusb/tmp/build\^" ^&^& cd libusb ^&^& ls ^&^& bootstrap.sh ^"
ERROR: libusb/1.0.0@PROJECT: Error in build() method, line 19
        tools.run_in_windows_bash(self, "cd libusb && ls && bootstrap.sh")
        TypeError: __call__() got an unexpected keyword argument 'win_bash'
```
This appears to be in error according to the documentation.
My conanfile.py file follows:
```
from conans import ConanFile, AutoToolsBuildEnvironment, tools

class LibusbConan(ConanFile):
    name = "libusb"
    version = "1.0.0"
    license = "LGPL 2.1"
    url = "git@github.com:leapmotion/libusb.git"
    description = "<Description of Libusb here>"
    settings = "os", "compiler", "build_type", "arch"
    options = {"shared": [True, False]}
    default_options = "shared=True"
    generators = "cmake"

    def source(self):
        self.run("git clone git@github.com:leapmotion/libusb.git")
        self.run("cd libusb && git checkout fix-soname-0")

    def build(self):
        tools.run_in_windows_bash(self, "cd libusb && ./bootstrap.sh")
        autotools = AutoToolsBuildEnvironment(self)
        autotools.configure(args=["--disable-udev"])
        autotools.make()

    def package(self):
        self.copy("*.h", dst="include", src="hello")
        self.copy("*hello.lib", dst="lib", keep_path=False)
        self.copy("*.dll", dst="bin", keep_path=False)
        self.copy("*.so", dst="lib", keep_path=False)
        self.copy("*.dylib", dst="lib", keep_path=False)
        self.copy("*.a", dst="lib", keep_path=False)

    def package_info(self):
        self.cpp_info.libs = ["hello"]
```

To help us debug your issue please explain:

- [x] I've read the [CONTRIBUTING guide](https://raw.githubusercontent.com/conan-io/conan/develop/.github/CONTRIBUTING.md).
- [x] I've specified the Conan version, operating system version and any tool that can be relevant.
- [x] I've explained the steps to reproduce the error or the motivation/use case of the question/suggestion.

**Comment by yeswalrus** [comment]
I just noticed this is a duplicate of https://github.com/conan-io/conan/issues/2850.

**Comment by memsharded** [comment]
Yes, this will be fixed and released in 1.3.2 soon, thanks for reporting.
